app/services/analytics_events.rb

```diff
@@ -226,9 +226,8 @@ def backup_code_created(enabled_mfa_methods_count:, **extra)
   end
 
   # Tracks when the user visits the Backup Code Regenerate page.
-  # @param [String] request_came_from the controller/action the request came from
-  def backup_code_regenerate_visit(request_came_from:, **extra)
-    track_event('Backup Code Regenerate Visited', request_came_from:, **extra)
+  def backup_code_regenerate_visit(**extra)
+    track_event('Backup Code Regenerate Visited', **extra)
```

Comment:
This might have been a merge issue with #8985. I wouldn't expect to see this change.

```suggestion
  # @param [String] request_came_from the controller/action the request came from
  def backup_code_regenerate_visit(request_came_from:, **extra)
    track_event('Backup Code Regenerate Visited', request_came_from:, **extra)
```

Reply:
I thought we talked about getting rid of the referrer implementation for this format instead or do we want ot keep the referrer for now.

Reply:
Oh, you're right. This looked to me like an issue with a merge conflict from #8985, but I see we're updating the call site to pass the new parameter.

Can we add the named parameters to the analytics methods where we're now receiving `in_multi_mfa_selection_flow` ?